### DISCUSSION TITLE
Faster Gaussian Splats

### DISCUSSION THREAD

**Discussion by draggett** [gpuweb/gpuweb#6249] [Ideas/Feedback]
Gaussian Spats are rapidly replacing textured triangles as the basis for rendering high fidelity 3D scenes, and moreover are better suited to AI due to the intrinsic differentiability as required for backpropagation and gradient descent. One challenge is hardware accelerated sorting of many millions of splats, another is speeding the rendering of individual splats.  How can WebGPU address these challenges?

**Comment by draggett** [comment]
One consideration is how to enable optimal use of memory local to the GPU to speed tile-based index sorting.  Another is to expose kernels that can efficiently render splats with the Gaussian drop off from the spat's center.  Having to translate spats into triangles is sub-optimal! Some motivating web sites: https://superspl.at and https://superspl.at/scene/ac0acb0e
